packages/communication-react/review/communication-react.api.md

```diff
@@ -1091,7 +1101,11 @@ export const DEFAULT_COMPOSITE_ICONS: {
     OptionsCamera: JSX.Element;
     OptionsMic: JSX.Element;
     OptionsSpeaker: JSX.Element;
-    ParticipantItemMicOff: JSX.Element;
+    ParticipantItemMicOff: JSX.Element; /**
+    * Icons that can be overridden for {@link CallComposite}.
```

Comment:
This comment seems to be in a weird spot.

Reply:
This is super strange, it's not related to this prop at all and this file isn't supposed to have block comments -- but manually removing this and rebuilding locally adds it back in (checking in CI now). Looks like an api-extractor bug but trying to figure out what in this PR has caused it...

Reply:
OK is it more good or less good that my local build was the one introducing this.. might be time to nuke the local clone